modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-vm-suma.adoc

```diff
@@ -17,16 +17,6 @@ Images for {productname} {productnumber} are available at link:https://www.suse.
 | s390x | qcow2, raw
 |===
 
-.Available Proxy Images
-[cols="3, 3", options="header"]
-|===
-| Architecture | Image Format
-
-| aarch64| qcow2, vmdk, raw
-
-| x86_64  | qcow2, vmdk, raw, Self Installer
-
-|===
 
```

Comment:
We should add a note here that raw images preparation https://documentation.suse.com/en-us/sle-micro/5.5/single-html/SLE-Micro-deployment/#sec-raw-preparation and deployment https://documentation.suse.com/en-us/sle-micro/5.5/single-html/SLE-Micro-deployment/#cha-images-procedure and same for the self install images https://documentation.suse.com/en-us/sle-micro/5.5/single-html/SLE-Micro-deployment/#cha-selfinstal-procedure.

We should do this both for the server and proxy docs.